src/web/lib/withSignInGateSlot.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+// This span is used to insert the sign in gate into the appropirate location within body of an article,
+// if the SignInGateSector determines a gate should be rendered.
+
+const SignInGateSlot = <span id="sign-in-gate" />;
+
+export const withSignInGateSlot = (
```

Comment:
Looks ok to me, just pinging @nicl for awareness of the method

Reply:
thanks!